makefile.example

```diff
@@ -0,0 +1,104 @@
+# migrations placed under this directory
+# do not edit this section
+MIGRATION_DIR=./migrations
+
+# edit this to match your env dev|prod|test etc
+ENV=dev
+
+# ==========================================================
+# edit this fields to match your database enviroment
+# must match with app database
+# user name of database
+POSTGRES_USER=edunota
+
+# pasword of database
+POSTGRES_PASSWORD=edunota
+
+# database to apply migrations
+POSTGRES_DB=edunota
+
+# port of database
+POSTGRES_PORT=5432
+
+# hostname of database
+POSTGRES_HOST=localhost
+
+
+# ============================================
+# used by goose, do not edit these fields																								    
+export GOOSE_DBSTRING=user=$(POSTGRES_USER) password=$(POSTGRES_PASSWORD) host=$(POSTGRES_HOST) port=$(POSTGRES_PORT) dbname=$(POSTGRES_DB)
```

Comment:
2023/09/17 22:12:35 goose run: failed to ensure DB version: failed to connect to `host=localhost user=edunota database=edunota`: failed SASL auth (FATAL: password authentication failed for user "edunota" (SQLSTATE 28P01))